python/semantic_kernel/core_plugins/conversation_summary_plugin.py

```diff
@@ -30,17 +31,21 @@ def __init__(
     ) -> None:
         """Initializes a new instance of the ConversationSummaryPlugin class.
 
-        :param kernel: The kernel instance.
-        :param prompt_template_config: The prompt template configuration.
-        :param return_key: The key to use for the return value.
+        Args:
+            kernel (Kernel): The kernel instance to use for the function.
+            prompt_template_config (PromptTemplateConfig): The configuration to use for functions.
+            return_key (str): The key to use for the return value.
         """
         self.return_key = return_key
-        self._summarizeConversationFunction = kernel.add_function(
+        kernel.add_function(
             prompt=ConversationSummaryPlugin._summarize_conversation_prompt_template,
             plugin_name=ConversationSummaryPlugin.__name__,
             function_name="SummarizeConversation",
             prompt_template_config=prompt_template_config,
         )
+        self._summarizeConversationFunction: "KernelFunction" = kernel.get_function(
```

Comment:
Do we need the 2 steps to get a function? Seems redundant to have to add and get it when the add could just return it.